apps/envConstants.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
   DRONE: process.env.DRONE,
   CIRCLE_TEST_REPORTS: process.env.CIRCLE_TEST_REPORTS,
   // If set, run webpack dev server (with hot reloading)
-  HOT: !!process.env.HOT,
+  HOT: process.env.HOT === '1',
```

Comment:
Gotta run home right now, but will this be false in our Gruntfile dev setup because we set process.env.HOT to 1 (number)? 🤔 Can check a bit later